src/shared/components/cells/DashboardCells.tsx

```diff
@@ -33,37 +41,47 @@ type ReduxProps = ConnectedProps<typeof connector>
 type Props = OwnProps & ReduxProps
 
 @ErrorHandling
-class Cells extends Component<Props> {
+class DashboardCellsUnconnected extends Component<Props> {
   public render() {
-    const {views, cells, manualRefresh} = this.props
+    const {cells, manualRefresh, status, views} = this.props
 
     return (
-      <>
-        <ScrollDetector component="dashboard" />
-        <Grid
-          cols={12}
-          layout={this.cells}
-          rowHeight={DASHBOARD_LAYOUT_ROW_HEIGHT}
-          useCSSTransforms={false}
-          containerPadding={[0, 0]}
-          margin={[LAYOUT_MARGIN, LAYOUT_MARGIN]}
-          onLayoutChange={this.handleLayoutChange}
-          draggableHandle=".cell--draggable"
-          isDraggable
-          isResizable
-        >
-          {fastMap(cells, cell => (
-            <div
-              key={cell.id}
-              className="cell"
-              data-testid={`cell ${views[cell.id]?.name}`}
-            >
-              <CellComponent cell={cell} manualRefresh={manualRefresh} />
-              <GradientBorder />
-            </div>
-          ))}
-        </Grid>
-      </>
+      <SpinnerContainer loading={status} spinnerComponent={<TechnoSpinner />}>
+        <Page.Contents fullWidth={true} scrollable={true} className="dashboard">
```

Comment:
The spinner is a nice add - what's the reason for adding `Page.Contents`?

Reply:
the spinner and `Page.Contents` were moved over from [`Dashboard.tsx`](https://github.com/influxdata/ui/blob/master/src/dashboards/components/Dashboard.tsx)

Reply:
That's what I get for not expanding the Big Red Diff in that file.